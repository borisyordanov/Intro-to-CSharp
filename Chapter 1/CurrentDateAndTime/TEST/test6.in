1691
3
13
17
3
0
