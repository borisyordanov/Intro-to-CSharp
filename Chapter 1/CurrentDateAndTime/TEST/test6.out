3/13/1691 5:03:00 PM
